user/models.py

```diff
@@ -10,6 +11,23 @@
 from trips.models import UserLocation
 
 
+def uniquify_username(username):
+    try:
+        # We stop when the exception is caught
+        while True:
+            str_number = re.match('.*?([0-9]*)$', username).group(1)
```

Comment:
With this solution, it will always add 1 to `username` because the regex match with `(.*)` and the digit at the end are included in `(.*)`
However I found that the code is good :+1: